src/controllers/authController.js

```diff
@@ -31,10 +36,14 @@ const createUser = async (req, res) => {
     });
 
     await user.save();
-    req.flash("alert", `User ${user_name} successfully signed up.`);
+    req.flash("success", `User ${user_name} successfully signed up.`);
     res.redirect("/auth/login");
   } catch (error) {
+    if (error.code === 11000) {
```

Comment:
why is the error code 11000? 
I don't think this is a valid error code.
Could you please change to a valid one?